### PR TITLE
Timer: moving schedule logging to Info

### PR DESCRIPTION
**Pull request by brettsam**
Addresses #37

### CODE REVIEW

**Comment by mathewc** [comment]
:shipit:

File: src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs [hunk 1]
```diff
@@ -112,7 +112,7 @@ internal System.Timers.Timer Timer
 
             // log the next several occurrences to console for visibility
             string nextOccurrences = TimerInfo.FormatNextOccurrences(_schedule, 5);
-            _trace.Verbose(nextOccurrences);
+            _trace.Info(nextOccurrences, "WebJobs.Execution");
```
Comment (mathewc) [inline src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs]: Yeah, I remember this issue now, which is why I haven't addressed this yet.

Comment (brettsam) [inline src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs]: New commit now has the latest from Core. I verified the schedule shows up with Info now.

File: src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs [hunk 2]
```diff
@@ -112,7 +112,7 @@ internal System.Timers.Timer Timer
 
             // log the next several occurrences to console for visibility
             string nextOccurrences = TimerInfo.FormatNextOccurrences(_schedule, 5);
-            _trace.Verbose(nextOccurrences);
+            _trace.Info(nextOccurrences);
```
Comment (mathewc) [inline src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs]: 👍

**Comment by mathewc** [comment]
:ship:

**Pull request merged**